gorm/README.md

```diff
@@ -27,3 +27,44 @@ db.password=dbpassword
 
 ```
 
+
+# Controller
+
+```
+type GormController struct {
+	revel.Controller
```

Comment:
The readme specifies how it works. Not how to use it

The way of use is to do the following:

```
type MyController struct {
    gorm.GormController
}
```